src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+@use "../utils/functions" as *;
+@use "../utils/variables" as *;
+@use "../utils/placeholders";
+
+.calendar {
+  @extend %center;
+  height: 100vh;
+
+  &__row {
+    display: flex;
+    flex-wrap: wrap;
+    width: 750px;
```

Comment:
You have forgotten about the gap, also consider it when calculating the margins for moving the first day of the month